pkg/testutil/kopia_testutil.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2023 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutil
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"os"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	awsconfig "github.com/kanisterio/kanister/pkg/aws"
+)
+
+const (
+	DefaultRepositoryPath                 = "kopia-repo-controller-test"
+	DefaultRepositoryServerAdminUser      = "admin@test"
+	DefaultRepositoryServerHost           = "localhost"
+	DefaultRepositoryPassword             = "test1234"
+	DefaultKanisterAdminUser              = "kanisterAdmin"
+	DefaultKanisterUser                   = "kanisteruser"
+	DefaultKanisterNamespace              = "kanister"
+	DefaultKopiaRepositoryServerContainer = "repo-server-container"
+	ServerReady                           = "ServerReady"
+)
```

Comment:
Can you please define what each of the constants here are?

Reply:
Will do that